Restructure GCP and Kube-resilience faults (#680)
harness/developer-hub: docs/chaos-engineering/chaos-faults/gcp/gcp-vm-disk-loss-by-label.md

![GCP VM Disk Loss By Label](./static/images/gcp-vm-disk-loss-by-label.png)

## Use cases

> - GCP VM disk loss by label fault can be used to determine the resilience of the GKE infrastructure. 
> - It helps determine how quickly a node can recover when a persistent disk volume is detached from the VM instance associated with it.

Reviewer: ```suggestion
- GCP VM disk loss by label fault can be used to determine the resilience of the GKE infrastructure. It helps determine how quickly a node can recover when a persistent disk volume is detached from the VM instance associated with it.
```

Reviewer: I think one bullet is fine since starting a second bullet with "It" seems odd.

      </tr>
      <tr>
        <td> GCP_PROJECT_ID </td>

>         <td> ID of the GCP project, of which the disk volumes are a part. </td>

Reviewer: ```suggestion
        <td> Id of the GCP project containing the disk volumes. </td>
```

Reviewer: ID = Identity Document.
Id = Identifier. Always use Id.
